Shell/Options.cpp

```diff
@@ -179,6 +181,11 @@ void Options::init()
     _lookup.insert(&_slowness);
     _slowness.onlyUsefulWith(UsingPortfolioTechnology());
 
+    _randomizedPortfolioWorkers = BoolOptionValue("randomized_portfolio_workers","",true);
+    _randomizedPortfolioWorkers.description = "In portfolio mode, let each worker process start from its own independent random seed.";
+    _lookup.insert(&_randomizedPortfolioWorkers);
+    _randomizedPortfolioWorkers.onlyUsefulWith(UsingPortfolioTechnology());
```

Comment:
No issue here, I'm just really pleased with how the option constraint stuff is going, this line is really beautiful!